helm/kubernetes-cluster-prep/README.md

```diff
@@ -182,6 +182,18 @@ The steps are as follows:
    -----END CERTIFICATE-----
    ```
 
+   The script will display the fingerprint for the certificate and ask if
+   you trust the the certificate. The fingerprint can be verified by running
+   the following command on the conjur server.
+   ```
+   openssl x509 -fingerprint -noout -in <certificate file path>
+   ```
+   The certificate is located at ~conjur/etc/ssl/conjur.pem
+
+   In a Kubernetes cluster, the certificate is located at
```

Comment:
Maybe we can be more explicit about when this case applies.
Do we know if this case applies to both Conjur Open Source and decomposed follower?
I don't know the answer to this offhand, but someone in conjur-core would know off the top.

For this case we can provide a helpful example `kubectl` command.